zio-http/src/main/scala/zhttp/service/client/content/handlers/DynamicContentHandler.scala

```diff
@@ -0,0 +1,39 @@
+package zhttp.service.client.content.handlers
+
+import io.netty.buffer.{ByteBuf, Unpooled}
+import io.netty.channel.{ChannelHandlerContext, SimpleChannelInboundHandler}
+import io.netty.handler.codec.http.{FullHttpResponse, HttpHeaderNames}
+import zhttp.http.{HTTP_CHARSET, Header, Status}
+import zhttp.service.Client
+import zhttp.service.Client.{ClientResponse, StringContent}
+
+final case class DynamicContentHandler() extends SimpleChannelInboundHandler[FullHttpResponse](true) {
```

Comment:
```suggestion
final class DynamicContentHandler() extends SimpleChannelInboundHandler[FullHttpResponse](true) {
```

As a general practice, we don't use case classes unless its immutable.

Reply:
updated.